csaf_2.0/examples/csaf/rhsa-2021_5186.json

```diff
@@ -0,0 +1,386 @@
+{
+  "document": {
+    "aggregate_severity": {
+      "namespace": "https://access.redhat.com/security/updates/classification/",
+      "text": "Critical"
+    },
+    "category": "security_advisory",
+    "csaf_version": "2.0",
+    "distribution": {
+      "text": "Copyright \u00a9 2022 Red Hat, Inc. All rights reserved.",
+      "tlp": {
+        "label": "WHITE",
+        "url": "https://www.first.org/tlp/"
+      }
+    },
+    "lang": "en",
```

Comment:
I left this off on purpose since Red Hat Product Security has associates in Australia and other commonwealth countries, India, the US, etc. I don't think we consistently use / enforce only US spelling.